tests/defer/src/jsTest/kotlin/test/DeferWithHelixTest.kt

```diff
@@ -26,19 +26,21 @@ import helix.HelixServer
 import kotlinx.coroutines.flow.toList
 import util.dynamicObject
 import util.jsAsyncIterator
+import kotlin.random.Random
 import kotlin.test.Ignore
 import kotlin.test.Test
 import kotlin.test.assertEquals
 
 @Ignore
-// TODO These tests are temporarily ignored while @defer has been renamed to @experimental_defer
-// See https://github.com/apollographql/apollo-kotlin/issues/4071
+// TODO These tests are temporarily ignored as the latest version of graphql-js
+// uses a package format which is incompatible with Kotlin/JS.
+// See https://youtrack.jetbrains.com/issue/KT-12784
```

Comment:
Interestingly, this seems to be working for me. After removing the `@Ignore`, I get this:

```
$ ./gradlew -p tests :defer:jsNodeTest 

....

test.DeferWithHelixTest.queryWithFragmentSpreads PASSED
test.DeferWithHelixTest.queryWithInlineFragments PASSED
test.DeferWithHelixTest.subscriptionWithInlineFragment PASSED
test.DeferWithHelixTest.subscriptionWithFragmentSpreads PASSED
```

What am I missing?